pkg/pub_worker/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM gcr.io/google-containers/debian-base-amd64:v2.0.0
```

Comment:
We install Dart from zip anyways, so it's better to use a minimal image.

Reply:
Let's add a TODO to synchronize the two. I guess we will eventually download and run the analysis SDKs dynamically, so it will be only the main runtime that needs to be configured, and can be the same...